From 4d55d6921e4e1c5322ffb430f11128cd1ce44ab0 Mon Sep 17 00:00:00 2001
From: Dong Aisheng <b29396@freescale.com>
Date: Fri, 15 May 2015 14:29:50 +0800
Subject: [PATCH] MLK-10837 dts: imx_v7_defconfig: remove CONFIG_BT_HCIBTSDIO
 support

We don't need btsdio driver support and enable it will cause
Broadcom WiFi driver suspend/resume unwork due to it does not
implement suspend/resume callback.

Signed-off-by: Dong Aisheng <b29396@freescale.com>
---
 arch/arm/configs/imx_v7_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/configs/imx_v7_defconfig b/arch/arm/configs/imx_v7_defconfig
index f940021..8e80173 100644
--- a/arch/arm/configs/imx_v7_defconfig
+++ b/arch/arm/configs/imx_v7_defconfig
@@ -80,7 +80,6 @@ CONFIG_BT_BNEP_MC_FILTER=y
 CONFIG_BT_BNEP_PROTO_FILTER=y
 CONFIG_BT_HIDP=y
 CONFIG_BT_HCIBTUSB=y
-CONFIG_BT_HCIBTSDIO=y
 CONFIG_BT_HCIUART=y
 CONFIG_BT_HCIUART_H4=y
 CONFIG_BT_HCIUART_BCSP=y
-- 
1.8.0

